pkg/services/replicator/process.go

```diff
@@ -45,6 +49,11 @@ func (p *Replicator) HandleTask(ctx context.Context, task Task, res TaskResult)
 			stream = client.DemuxReplicatedObject(stream)
 		}
 	} else {
+		if err = p.checkExpiration(task.obj); err != nil {
+			p.log.Debug("expiration check", zap.Error(err))
+			return
```

Comment:
iiuc `checkExpiration` eternal objects w/o parent are never replicated. I doubt this is a correct behavior

Reply:
fixed